### PR TITLE
feat: Start engine on the cloud instance during configuration

### PR DESCRIPTION
**Pull request by laurentluce**
We cannot force the user to use the CLI so we have to start the engine on the cloud instance ourselves.  We generate the contexts config and point the remote context to the cloud instance.  We use the CLI to start the engine.

### CODE REVIEW

File: contexts-config.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          "host": "{IP}",
+          "remotePortalPort": 9720,
+          "kurtosisBackendPort": 9722,
+          "tunnelPort": 9721,
+          "tlsConfig": {
+            "certificateAuthority": "{CA}",
+            "clientCertificate": "{CLIENT_CERT}",
+            "clientKey": "{CLIENT_KEY}"
+          },
+          "envVars": "{}"
```
Comment (laurentluce) [inline contexts-config.json]: The AWS env variables need to be added.

Comment (adschwartz) [reply contexts-config.json]: @laurentluce This is just for my curiosity, but what's the need to supply a context config? Is it not enough to intall kurtosis-cli and then run `kurtosis engine start`?  Ie. does this instance actually need the remote context config on the machine?

**Review by adschwartz — APPROVED** [review]
Looks good!

File: contexts-config.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "certificateAuthority": "{CA}",
+            "clientCertificate": "{CLIENT_CERT}",
+            "clientKey": "{CLIENT_KEY}"
+          },
+          "envVars": {
+            "AWS_ACCESS_KEY_ID": "{AWS_ACCESS_KEY_ID}",
+            "AWS_SECRET_ACCESS_KEY": "{AWS_SECRET_ACCESS_KEY}",
+            "AWS_BUCKET_REGION": "{AWS_BUCKET_REGION}",
+            "AWS_BUCKET_NAME": "{AWS_BUCKET_NAME}",
+            "AWS_BUCKET_USER_FOLDER": "{AWS_BUCKET_USER_FOLDER}"
```
Comment (adschwartz) [inline contexts-config.json:33]: @laurentluce this is specific to Hyperlane use case right? Unless there are new developments I haven't heard, I'm thinking that this is handled in a starlark script? cc @gbouv

Comment (gbouv) [reply contexts-config.json:33]: See my other comment https://github.com/kurtosis-tech/kurtosis-cloud-backend/pull/35/files#r1307342078 here

File: contexts-config.json (hunk 1 above)
Comment (adschwartz) [reply contexts-config.json]: Answered in other communicaton

**Pull request merged**